yaml/thz404.yaml

```diff
@@ -0,0 +1,7 @@
+esphome:
+  platformio_options:
+    build_flags:
+      - "-DTHZ_404"
+
+packages:
+  BASE: !include { file: wp_base.yaml, vars: { clk_pin: $clk_pin, mosi_pin: $mosi_pin, miso_pin: $miso_pin, cs_pin: $cs_pin, evu_pin: $evu_pin, entity_room_temperature: $entity_room_temperature, entity_humidity: $entity_humidity}}
```

Comment:
The pins should not be passed here. They are board specific and have nothing to do with the variant. We can think of a mechanism how he can support different boards later.

Reply:
This just fixed the mechnism which is already in the code:

```
defaults:
  clk_pin: "GPIO14"
  mosi_pin: "GPIO13"
  miso_pin: "GPIO4"
  cs_pin: "GPIO15"
  evu_pin: "GPIO5"
  entity_room_temperature: "sensor.durchschnittstemperatur_haus_ohne_keller"
  entity_humidity: "sensor.durchschnitt_luftfeuchtigkeit_haus"
```
To overrite the default pin config, the parameters needs to be passed on, so my currently used package include call is working again:
`thz55eco: !include { file: oneESPtoRuleThemAll/yaml/thz5_5_eco.yaml,  vars: { clk_pin: "GPIO0", mosi_pin: "GPIO16", miso_pin: "GPIO39", cs_pin: "GPIO33", evu_pin: "GPIO15", entity_room_temperature: "sensor.cr_controller_kitchen_temp_dining", entity_humidity: "sensor.humidity_central"}}`

Reply:
How about using 
substitutions:
  clk_pin: "GPIO14"
  mosi_pin: "GPIO13"
  miso_pin: "GPIO4"
  cs_pin: "GPIO15"
  evu_pin: "GPIO5"
  entity_room_temperature: "sensor.durchschnittstemperatur_haus_ohne_keller"
  entity_humidity: "sensor.durchschnitt_luftfeuchtigkeit_haus"
  
  those could be included on the highest level and change the pin assignment?

Reply:
thanks, changed this as proposed to substitution